scripts/check_type_hint_coverage.py

```diff
@@ -3,7 +3,7 @@
 from typing import Dict, List, Optional
 
 TYPE_HINT_ERROR_THRESHOLD: int = (
-    2529  # This number is to be reduced as we annotate more functions!
+    2533  # This number is to be reduced as we annotate more functions!
```

Comment:
I'd like to push back on increasing this number for this PR. Is there a way to proceed without adjusting this number @alexsherstinsky ?

Reply:
Trying to get it back to the previous number.